openshift-ci/Dockerfile.bundle-4.7.ci

```diff
@@ -0,0 +1,15 @@
+# see: https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md
+FROM scratch
+
+ARG CHANNEL=4.7
```

Comment:
Do you want to promote the main dockerfile to 4.8? Or is it intended to be done in a separate PR?

Reply:
is meant to be on a separate PR. I'm trying to always keep CI happy, so first adding manifests we don't consume yet, then fixing openshift/release and lastly changing again PAO manifests